api/src/wfl/module/covid.clj

```diff
@@ -262,27 +263,28 @@
       flatten))
 
 (defn ^:private go-create-snapshot!
-  "Create snapshot in TDR from `dataset` body, `table`
-   and `row-ids` then write job info as well as rows
-   into `source-details-name` table, `suffix` will be
-   appended to the snapshot names."
-  [suffix dataset table row-ids]
+  "Create snapshot in TDR from `dataset` body, `table` and `row-ids` then
+   write job info as well as rows into `source-details-name` table.
+   Snapshots will be readable by members of the `snapshotReaders` list.
+   `suffix` will be appended to the snapshot names."
+  [suffix {:keys [dataset table snapshotReaders] :as _source} row-ids]
   (let [columns (->> (datarepo/all-columns dataset table)
                      (mapv :name)
                      (cons "datarepo_row_id"))]
     (-> (datarepo/make-snapshot-request dataset columns table row-ids)
         (update :name #(str % suffix))
+        (assoc :readers snapshotReaders)
```

Comment:
@tbl3rd

Reply:
But that's not in `create-snapshots` though this review UI obscures that.